readme.md

```diff
@@ -18,7 +18,6 @@ Our hope is that GitHub will notice and implement some of these much needed impr
 - [Adds user avatars to Reactions](screenshot-reactions.png)
 - [Adds a quick edit button to the readme](https://cloud.githubusercontent.com/assets/170270/13379292/61cd4c42-de54-11e5-8829-f4b82ba8c2bc.png)
 - [Shows current filename in the sticky pull request header](https://cloud.githubusercontent.com/assets/170270/14153322/97a8e902-f6e1-11e5-8331-19e284e3e6fa.png)
-- [Adds blame links for parent commits in blame view](https://github.com/sindresorhus/refined-github/issues/2#issuecomment-189141373)
```

Comment:
When we remove features GitHub implemented, I think we should instead strike `~~` and put it at the bottom with a comment that it was implemented.